src/scss/grommet-core/_objects.layer.scss

```diff
@@ -183,6 +183,16 @@
       }
     }
   }
+
+  .#{$grommet-namespace}tiles {
+    overflow: auto;
+
+    &__more {
+      @include media-query(lap-and-up) {
+        position: relative;
```

Comment:
Ah yes, I added the media query because the loading icon was being flushed to the left instead of staying in the middle, but setting `responsive={false}` to Tiles fixes this issue.